### PR TITLE
Portfolio: preserve data in the browser

### PR DESCRIPTION
**Pull request by SSEKPIUS**
I implemented the following interactions:

1. When the user changes the content of any input field, the data is saved to the local storage.
2. When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

### CODE REVIEW

**Review by AbhishekkM05 — COMMENTED** [review]
good javascript

**Comment by AbhishekkM05** [comment]
html , CSS , files missing

File: index.js [hunk 1]
```diff
@@ -248,4 +257,32 @@ form.addEventListener('submit', (event) => {
     error.querySelector('span').textContent = '';
     error.style.setProperty('display', 'none');
   }
+});
+// Intiate local storage for form data
+if (localStorage.getItem('form')) {
+  const data = JSON.parse(localStorage.getItem('form'));
+  firstName.value = data.firstName;
+  lastName.value = data.firstName;
```
Comment (FrederickMih) [inline index.js]: - [ ] When I enter the first name and reload the page, the input field for the last name is filled with the first name. This does not create a good user experience hence let's fix :wrench: it!



- [ ] When I fill in the first name input field, followed by the last name field then reload the page, the input field for the first name is not pre-filled with the data. Based on the requirements of this project, the input fields are pre-filled with the data. We shall kindly fix :wrench: this issue.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @SSEKPIUS, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Screenshot from 2022-10-20 09-38-51



## Highlights

- Descriptive README.md file :heavy_check_mark: 
- PR has a title and summary :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @SSEKPIUS, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**